library/std/src/lib.rs

```diff
@@ -32,9 +32,8 @@
 //!
 //! Once you are familiar with the contents of the standard library you may
 //! begin to find the verbosity of the prose distracting. At this stage in your
-//! development you may want to press the <code>
-//! <svg style="width:0.75rem;height:0.75rem" viewBox="0 0 12 12"
-//! stroke="currentColor" fill="none">
+//! development you may want to press the <code><svg style="width:0.75rem;height:0.75rem"
+//! viewBox="0 0 12 12" stroke="currentColor" fill="none">
 //! <path d="M2,2l4,4l4,-4M2,6l4,4l4,-4"/></svg> Summary</code> button near the
```

Comment:
does it format correctly if we just move all the HTML elements to the same line and then resume text on the next?

Reply:
Yes, although the line ends up quite long. I've pushed a commit showing it, and can revert it line length is an issue.

Reply:
Tidy seems to be okay with it.

...which is mildly surprising since it IS over 100 characters long...